unit_test/test_spdm_responder/vendor_response.c

```diff
@@ -0,0 +1,178 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2023-2024 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#include "spdm_unit_test.h"
+#include "internal/libspdm_responder_lib.h"
+
+#if LIBSPDM_ENABLE_VENDOR_DEFINED_MESSAGES
+
+typedef struct {
+    spdm_message_header_t header;
+    uint16_t standard_id;
+    uint8_t vendor_id_len;
+    uint8_t vendor_id[SPDM_MAX_VENDOR_ID_LENGTH];
+    uint16_t data_len;
+    uint8_t data[16];
+} libspdm_vendor_request_test;
+
+typedef struct {
+    spdm_message_header_t header;
+    uint16_t standard_id;
+    uint8_t vendor_id_len;
+    uint8_t vendor_id[SPDM_MAX_VENDOR_ID_LENGTH];
+    uint16_t data_len;
+    uint8_t data[64];
+} libspdm_vendor_response_test;
+#pragma pack(1)
```

Comment:
`#pragma pack(1)`
Here this is different with original c file. `#pragma pack()`